### PR TITLE
Switch to new LDSAuthentication pod for auth

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

File: LDSAnnotations/SyncAnnotationsOperation.swift [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 import Foundation
 import ProcedureKit
+import LDSAuthentication
```
Comment (SerenadeX) [inline LDSAnnotations/SyncAnnotationsOperation.swift]: Seems meaningless to add this and nothing else to this file

File: LDSAnnotations/SyncNotebooksOperation.swift [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 import Foundation
 import ProcedureKit
+import LDSAuthentication
```
Comment (SerenadeX) [inline LDSAnnotations/SyncNotebooksOperation.swift]: Seems meaningless to add this and nothing else to this file

File: LDSAnnotations/SyncAnnotationsOperation.swift (hunk 1 above)
Comment (stephanheilner) [inline LDSAnnotations/SyncAnnotationsOperation.swift]: It adds the AuthenticationOperation as a dependency operation. That class was moved from LDSAnnotations to LDSAuthentication which is why the only change that is needed is the import. But it uses the class that's in the LDSAuthentication framework now instead, so it needs it.

File: LDSAnnotations/SyncNotebooksOperation.swift (hunk 2 above)
Comment (stephanheilner) [inline LDSAnnotations/SyncNotebooksOperation.swift]: Same as above.

File: LDSAnnotations/SyncAnnotationsOperation.swift (hunk 1 above)
Comment (SerenadeX) [inline LDSAnnotations/SyncAnnotationsOperation.swift]: Sounds good to me

**Pull request merged**